pyvista/core/utilities/geometric_objects.py

```diff
@@ -39,6 +41,247 @@
 }
 
 
+@no_new_attr
+class ConeSource(_vtk.vtkConeSource):
+    """Create a cone source algorism class.
```

Comment:
typo (algorithm)

Reply:
Spelling fixed, and using nouns rather than verbs since this is a class.

```suggestion
    """Cone source algorithm class.
```

Reply:
Thanks!